### PR TITLE
fix: fee explainer modal close button

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

Fixes the dead click on the fee explainer close modal, likely caused by the Chakra package bump due to our incorrect usage (or lack thereof) of `<ModalBody>.

Screenshot 2024-08-23 at 10 57 22 AM

## Issue (if applicable)

Closes https://github.com/shapeshift/web/issues/7610

## Risk

> High Risk PRs Require 2 approvals

Low

> What protocols, transaction types, wallets or contract interactions might be affected by this PR?

None

## Testing

Ensure that the fee explainer can be closed using the close button (not just clicking the background).

### Engineering

☝️

### Operations

- [ ] :checkered_flag: My feature is behind a flag and doesn't require operations testing (yet)

☝️

## Screenshots (if applicable)

See above.

### CODE REVIEW

**Review by NeOMakinG — CHANGES_REQUESTED** [review]
Close but this is not the reason why it had a deadclick and it's not 100% fixed because half of the closing button is clickable

image

The main reason why it's a dead click is that the tab container is hover the closing button

Using a `ModalBody` almost fixes it because the body adds some paddings preventing the tablist to be hover the close button

I would keep the body but remove the padding so it's semantic is following chakra's idea and add a small zIndex on the closing button

File: src/components/FeeModal/FeeModal.tsx [hunk 1]
```diff
@@ -35,26 +36,28 @@ export const FeeModal = ({
       <ModalOverlay />
       <ModalContent>
         <ModalCloseButton />
```
Comment (NeOMakinG) [inline src/components/FeeModal/FeeModal.tsx:38]:
```suggestion
        <ModalCloseButton zIndex="1" />
```

File: src/components/FeeModal/FeeModal.tsx [hunk 2]
```diff
@@ -35,26 +36,28 @@ export const FeeModal = ({
...
-                inputAmountUsd={inputAmountUsd}
-                borderRadius='none'
-                bg='transparent'
-                boxShadow='none'
-                feeModel={feeModel}
-              />
-            </TabPanel>
-          </TabPanels>
-        </Tabs>
+        <ModalBody>
```
Comment (NeOMakinG) [inline src/components/FeeModal/FeeModal.tsx:39]:
```suggestion
        <ModalBody p={0}>
```

**Comment by 0xApotheosis** [comment]
That works, @NeOMakinG! Addressed in https://github.com/shapeshift/web/pull/7611/commits/42e5057695ba42687507b74c1b2a5462827fd9b7

The z-index does the trick of ensuring it stays clickable.

Screenshot 2024-08-24 at 8 46 09 AM

**Review by NeOMakinG — APPROVED** [review]
https://jam.dev/c/7267d48c-64e4-478a-9b35-bf58fe93430f

LGTM

**Pull request merged**